### ISSUE TITLE
multiple loss in R

### ISSUE THREAD

**Issue by ivokwee** [dmlc/mxnet#3437]
Hi. I am trying to get multiple loss function using R in this code 

data <- mx.symbol.Variable('data')
fc1  <- mx.symbol.FullyConnected(data = data, name = 'fc1', num_hidden = 64)
act1 <- mx.symbol.Activation(data = fc1, name = 'relu1', act_type = "relu")
fc2  <- mx.symbol.FullyConnected(data = act1, name = 'fc2', num_hidden = 10)

label1 <- mx.symbol.Variable('out1_label')
label2 <- mx.symbol.Variable('out2_label')
out1   <- mx.symbol.SoftmaxOutput(data = fc2, name='out1', label=label1)
out2   <- mx.symbol.SoftmaxOutput(data = fc2, name='out2', label=label2)
mlp    <- mx.symbol.Group(out1, out2)

Y <- list( out1_label=train.y, out2_label=train.y )

model <- mx.model.FeedForward.create(
    symbol             = mlp,
    X                  = t(train.x),
    y                  = Y,
    eval.metric        = mx.metric.rmse,
    array.layout = "colmajor"
)

But it get this  error:

Error in eval(substitute(expr), envir, enclos) : not compatible with requested type

Is multiple loss supported in the R interface? What might I be doing wrong?

Thanks, 

Ivo

**Comment by winstywang** [comment]
@thirdwing

**Comment by thirdwing** [comment]
I think this has not been supported in R side.

**Comment by Leloew** [comment]
Hey, any alternative to this? I am trying to implement an autoencoder and would like to have softmax loss for the categorical variablels and MAE for the numericall one.

**Comment by thirdwing** [comment]
@Leloew Can I know why you can't build two symbols according to two types of results?

**Comment by Leloew** [comment]
Hey @thirdwing, thanks for the reply. I want to have a shared hidden representaiotn for both the categorical and the numerical variables. Is it possible to implement this with two seperate symbols? or what do you mean exactly? 
Do you mean Symbol.group? I have read that it doesn't work for R in an other post.
